lib/web/imageRouter/index.js

```diff
@@ -15,12 +16,25 @@ const errors = require('../../errors')
 const imageRouter = (module.exports = Router())
 
 async function checkUploadType (filePath) {
-  const typeFromMagic = await FileType.fromFile(filePath)
+  const extension = path.extname(filePath).toLowerCase()
+  let typeFromMagic = await FileType.fromFile(filePath)
+  if (extension === '.svg') {
```

Comment:
Instead of the comparison of the uploaded extension, you could check whether file-info reported the file as xml and then do the is-svg check.

Reply:
We tried that and not all svgs are found as xml (only those that start with ´<xml>` do), but there are qute a few svgs that just start with `<svg>`. Those aren't identified by FileType at all. `typeFromMagic` is just `undefined`.

Reply:
Ah okay, then nevermind.